### ISSUE TITLE
After restart of Element new notifications show up in previously fully read rooms

### ISSUE THREAD

**Issue by kongo09** [vector-im/element-web#26360]
### Steps to reproduce

1. Read all rooms, no notifications left
2. Close Element
3. Restart computer
4. Open Element

#### What did you expect?
Still all rooms are read (if no new messages arrived in the meantime)

#### What happened instead?
Notification counter races up to 98 unread messages

### Operating system

Windows 11

### Homeserver

element.io

### Will you send logs?

Yes

**Comment by weeman1337** [comment]
Thank you for reporting the issue @kongo09 

I am going to close this in favour of https://github.com/vector-im/element-web/issues/24392 . Please keep an eye on the stuck notifications issue. If your problem still exists after #24392 was closed you can open another issue.
